raisinbread/instruments/bmi.linst

```diff
@@ -6,17 +6,20 @@ static{@}Window_Difference{@}Window Difference (+/- Days)
 select{@}Examiner{@}Examiner{@}NULL=>''
 static{@}{@}Note: Please enter measurements in Standard OR Metric units
 select{@}unit_classification{@}Choose unit of measurement{@}NULL=>''{-}'metric'=>'Metric'{-}'standard'=>'Standard'
+page{@}{@}Standard units
 static{@}{@}Standard units
 numeric{@}height_feet{@}Your height (feet) :
 select{@}height_feet_status{@}{@}NULL=>''{-}'not_answered'=>'Not Answered'
 numeric{@}height_inches{@}Your height (inches) :
 select{@}height_inches_status{@}{@}NULL=>''{-}'not_answered'=>'Not Answered'
-numeric{@}weight_lbs{@}Your weight (lbs) :
+numeric{@}weight_lbs{@}Your weight (lbs) :{@}50{@}500
```

Comment:
Is this required or it just because you assume no one weighs outside of that range? If it's just to fix warnings we should fix it in the code, because numeric without any max/min is valid (like on line 13..)

Reply:
it is valid, no warnings

I just try to keep diverse examples for different features in the RB dataset to catch failing feature when possible.